README.md

```diff
@@ -2,144 +2,56 @@
 
 A collection of composite GitHub Actions
 
-## Publish Build Scans® for pull-requests issued from forked repositories
+## Add a workflow summary to Maven builds 
 
 ### Description
-When submitting a pull request, a GitHub workflow that validates the change is usually triggered, however the Develocity Build Scans® can’t be published 
-as workflows from forked repositories do not have access to secrets although an access token is required to publish a Build Scan®
+The `maven-setup` action adds a workflow summary for each Maven invocation. 
+The summary can also be optionally added as a pull-request comment (if in a pull-request context). 
```

Comment:
If this is enabled by default, I would adjust the wording here.
```suggestion
The summary is also added as a pull-request comment in a pull-request context. This can be disabled via <link> 
```